flutter/lib/components/item.dart

```diff
@@ -2,22 +2,20 @@ import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:realm/realm.dart';
 import 'package:flutter_slidable/flutter_slidable.dart';
-import 'modify_todo.dart';
-import '../viewmodels/todo_viewmodel.dart';
+import 'modify_item.dart';
+import 'package:flutter_todo/viewmodels/item_viewmodel.dart';
 
-class TodoItem extends StatelessWidget {
-  final TodoViewModel viewModel;
+class ItemItem extends StatelessWidget {
+  final ItemViewModel viewModel;
   final Animation<double> animation;
 
-  const TodoItem(this.viewModel, this.animation, {Key? key}) : super(key: key);
+  const ItemItem(this.viewModel, this.animation, {Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     final realm = Provider.of<Realm>(context);
```

Comment:
Since we've moved the realm stuff to the viewModel, do we still need this line?

Reply:
see above comment. don't need this file at all 🙂